### ISSUE TITLE
About 3D Gaussians

### ISSUE THREAD

**Issue by Ysz2022** [buaacyw/MeshAnything#29]
Thanks for your nice work!

It is said that MeshAnuthing can extract mesh from 3D Gaussians well. But in Readme, I can only find two scripts, one is for Mesh (obj) and one is for point clouds (npy). How to use this method on 3D Gaussians (ply)?

Hoping for your reply.

Thanks!

**Comment by buaacyw** [comment]
We suggest converting the 3D Gaussians into a mesh before inputting them into our method.

**Comment by Dumeowmeow** [comment]
> We suggest converting the 3D Gaussians into a mesh before inputting them into our method.

Why not just convert to a point cloud, don't 3D Gaussians have xyz properties?

**Comment by buaacyw** [comment]
Yes. But directly using the xyz from 3D Gaussians can't represent the shape of the 3D Gassuains.
